ios/scripts/upload-testflight.sh

```
@@ -127,6 +127,111 @@ verify_ipa_bundle_identity() {
+current = info.get("MinimumOSVersion")
+if isinstance(current, str) and current.strip():
+    raise SystemExit(0)
+
+info["MinimumOSVersion"] = minimum_os
+with open(plist_path, "wb") as handle:
+    plistlib.dump(info, handle)
+print("1")
+PY
+)"
+    if [[ "$changed" == "1" ]]; then
+      echo "stamped MinimumOSVersion=$minimum_os on nested framework: ${framework_info#$app/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)